Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN apt-get update && apt-get install -y \
 	automake \
 	bash-completion \
 	btrfs-tools \
+        bridge-utils \
```

Comment:
`iproute2` satisfies too and it is already in testing image, no need to bust cache.